src/modules/NetworkLogger.ts

```diff
@@ -16,7 +16,8 @@ let _requestFilterExpression = 'false';
 export const setEnabled = (isEnabled: boolean) => {
   if (isEnabled) {
     xhr.enableInterception();
-    xhr.setOnDoneCallback(network => {
+    xhr.setOnDoneCallback((network) => {
+      // eslint-disable-next-line no-new-func
       const predicate = Function('network', 'return ' + _requestFilterExpression);
```

Comment:
Unfortunately, it's not possible because `_requestFilterExpression` is a string not a predicate. A change was introduced in #708 but it was closed until it gets planned. So it's expected to get fixed as soon as we introduce that change.